tests/kernels/test_cutlass.py

```diff
@@ -507,3 +508,136 @@ def test_cutlass_cuda_graph(per_act_token: bool, per_out_ch: bool):
 
 def test_cutlass_support_opcheck():
     opcheck(torch.ops._C.cutlass_scaled_mm_supports_fp8, (capability, ))
+
+
+# TODO add bias
+@pytest.mark.parametrize("num_groups", [8, 64])
+@pytest.mark.parametrize("per_act_token", [True, False])
+@pytest.mark.parametrize("per_out_ch", [True, False])
+@pytest.mark.parametrize("use_bias", [False])
+@pytest.mark.skipif(not current_platform.has_device_capability(89),
```

Comment:
use `current_platform.supports_fp8`

Reply:
I think this actually needs to test for exactly `sm90a`. Can query `check_cutlass_group_gemm_supported` for this. But need to skip the test on RoCM as well here, so `current_platform.supports_fp8` won't be sufficient

Reply:
I added function `cutlass_group_gemm_supported`, can you have a look?